src/main/scala/com/microsoft/hyperspace/actions/OptimizeAction.scala

```diff
@@ -141,18 +141,16 @@ class OptimizeAction(
     // the list of newly created index files.
     val hadoopConf = spark.sessionState.newHadoopConf()
     val absolutePath = PathUtils.makeAbsolute(indexDataPath.toString, hadoopConf)
-    val newContent =
-      Content.fromDirectory(absolutePath, fileIdTracker, hadoopConfiguration = hadoopConf)
+    val newContent = Content.fromDirectory(absolutePath, fileIdTracker, hadoopConf)
     val updatedDerivedDataset = previousIndexLogEntry.derivedDataset.copy(
       properties = previousIndexLogEntry.derivedDataset.properties
         .copy(
-          properties =
-              Hyperspace
-                .getContext(spark)
-                .sourceProviderManager
-                .enrichIndexProperties(
-                  previousIndexLogEntry.relations.head,
-                  prevIndexProperties + (IndexConstants.INDEX_LOG_VERSION -> endId.toString))))
+          properties = Hyperspace
```

Comment:
scalafmt changed this block. Since this PR is small enough, I let this change in.